runhouse/main.py

```diff
@@ -1092,3 +820,90 @@ def main(
         print(f"{__version__}")
     elif ctx.invoked_subcommand is None:
         subprocess.run("runhouse --help", shell=True)
+
+
+# TODO [SB/JL]: uncomment and update once Rohin's change is merged.
+# TODO [SB/JL]: once supported, need to add cluster_logs to cli.rst
+# @cluster_app.command("logs")
+# def cluster_logs(
+#     cluster_name: str = typer.Argument(
```

Comment:
we can save this logic elsewhere outside this PR, for the time being can we just replace with a "pass"?